components/application-authenticators/org.wso2.carbon.identity.application.authenticator.requestpath.oauth/src/main/java/org/wso2/carbon/identity/application/authenticator/requestpath/oauth/OAuthRequestPathAuthenticator.java

```diff
@@ -44,6 +44,7 @@
     private static final String AUTHORIZATION_HEADER_NAME = "Authorization";
     private static final String BEARER_SCHEMA = "Bearer";
     private static final String AUTHENTICATOR_NAME = "OAuthRequestPathAuthenticator";
+    private static final String ACCESS_TOKEN = "access_token";
```

Comment:
Needs to add a new dependency for Oltu to get this constant. Hence went ahead with introducing a private variable than introducing the dependency